ultralytics/cfg/__init__.py

```diff
@@ -185,6 +184,9 @@
     "profile",
     "multi_scale",
 }
+DEPRECATED_KEYS = {
```

Comment:
Just curious: Why was it included in this set? Usually, deprecated things are removed at the next minor version, do we use this standard? If so, we need to update the minor version.